test/metabase/upload_test.clj

```diff
@@ -412,7 +412,11 @@
     (testing "File name is slugified"
       (is (=? #"my_file_name_\d+" (@#'upload/unique-table-name driver/*driver* "my file name"))))
     (testing "semicolons are removed"
-      (is (nil? (re-find #";" (@#'upload/unique-table-name driver/*driver* "some text; -- DROP TABLE.csv")))))))
+      (is (nil? (re-find #";" (@#'upload/unique-table-name driver/*driver* "some text; -- DROP TABLE.csv")))))
+    (testing "No collisions"
+      (let [n 50
+            names (repeatedly n (partial #'upload/unique-table-name driver/*driver* ""))]
+        (is (= 50 (count (distinct names))))))))
```

Comment:
There should be a test for what would've been a timestamp collision. There are test helpers to help fake the clock if that's useful.

Reply:
I'm not sure I follow - this test is totally sufficient in my opinion. We want to assert that the within tests it is not necessary to call `sleep` between calling any methods that will rely on the table name, and the repetition and direct low level call give us strong confidence.

Were you imagining testing a higher level workflow? In my mind that's unnecessary - the "proof" is the lack of test flakes or slow tests. 

I'm not keen on faking the clock, that feels quite invasive. This test already fails without the changes and I feel it is sufficient. Have I swayed you?

Reply:
Sorry, that's my bad